readme.md

```diff
@@ -58,6 +60,8 @@ A **class** is a blueprint from which objects are made. In javascript we used cl
 
 Let's define a `User` class. We'll be using `binding.pry` to test our code.
 
+Aside: pry is a ruby gem that allows us to work with ruby code in an IRB (interactive ruby shell). It's similar to working in our developer console in a web browser with javascript.
```

Comment:
I like putting this kind of information in a footnote. You can do that in markdown by preceding it with a `>`, so that it looks like this...

> Pry is a ruby gem that allows us to work with ruby code in an IRB (interactive ruby shell). It's similar to working in our developer console in a web browser with javascript.